network.py

```diff
@@ -88,6 +88,31 @@ def create_vgglike_network(input_shape):
     return Model(input, x)
 
 
+def create_inception_network(input_shape):
+    """
+       Simple architecture with one layer of inception model
+
+       param input_shape: shape of the input image
+    """
+
+    input = Input(shape=input_shape)
+
+    x1 = Conv2D(64, (1, 1), activation='relu', padding='same')(input)
+    x1 = Conv2D(64, (3, 3), activation='relu', padding='same')(x1)
+
+    x2 = Conv2D(64, (1, 1), activation='relu', padding='same')(input)
+    x2 = Conv2D(64, (5, 5), activation='relu', padding='same')(x2)
+
+    x3 = MaxPooling2D((3, 3), strides=(1, 1), padding='same')(input)
+    x3 = Conv2D(64, (1, 1), activation='relu', padding='same')(x3)
+
+    x = concatenate([x1, x2, x3], axis=3)
+    x = Flatten()(x)
+    x = Dense(2, activation='softmax')(x)
```

Comment:
We probably don't need the softmax layer here because of the architecture we're using.
It might be worth retesting with a Dense layer instead (like in the `vgg16` or `vgglike` cases).